src/Lecture2.hs

```diff
@@ -70,7 +78,11 @@ return the removed element.
 >>> removeAt 10 [1 .. 5]
 (Nothing,[1,2,3,4,5])
 -}
-removeAt = error "TODO"
+removeAt :: Int -> [Int] -> (Maybe Int, [Int])
```

Comment:
I see that tests don't compile. This is because the type signature of this function is more specific than it could be 🙂 
```suggestion
removeAt :: Int -> [a] -> (Maybe a, [a])
```